src/lib/menu/README.md

```diff
@@ -12,24 +12,10 @@
 ## Usage
 
 ### Setup
-
-Import the MdMenu module.
-
-*my-app-module.ts*
-```ts
-import {MdMenuModule} from '@angular2-material/menu';
-
-@NgModule({
-  imports: [MdMenuModule.forRoot()],
-  ...
-})
-export class MyAppModule {}
-```
-
 For alpha.7, you need to include the overlay styles in your app via a `link` element. This will
 look something like
 ```html
-<link href="vendor/@angular2-material/core/overlay/overlay.css" rel="stylesheet">
+<link href="vendor/@angular/material/core/overlay/overlay.css" rel="stylesheet">
```

Comment:
This whole blurb about including the overlay styles can actually go away now.

(same for tooltip readme)